core/primitives/src/stateless_validation.rs

```diff
@@ -102,10 +175,12 @@ pub struct ChunkStateWitness {
     /// with chunk_header.prev_block_hash().
     /// This is needed to validate signature when the previous block is not yet
     /// available on the validator side (aka orphan state witness).
+    /// TODO(stateless_validation): Deprecate once we send state witness in parts.
```

Comment:
can you use a issue id #ID instead of stateless_validation unless it is already common to use this keyword for TODOs? for finding out remaining tasks in code for a specific issue.

Reply:
Yes sure! Let me update and put the issue ID. Fwiw, this is just a short term thing till I get the full implementation in place.